commerce_coordinator/apps/commercetools/sub_messages/tasks.py

```diff
@@ -160,23 +161,23 @@ def fulfill_order_sanctioned_message_signal_task(
         return False
 
     if not (customer and order and is_edx_lms_order(order)):
-        logger.debug(f'[CT-{tag}] order %s is not an edX order', order_id)
+        logger.info(f'[CT-{tag}] order %s is not an edX order', order_id)
         return True
 
     if get_edx_is_sanctioned(order):
-        logger.debug(
+        lms_user_name = get_edx_lms_user_name(customer)
+        logger.info(f'[CT-{tag}] calling lms to deactivate user %s', lms_user_name)
+
+        LMSAPIClient().deactivate_user(lms_user_name)
```

Comment:
is there a reason this isnt yet another task and signal? (im fine either way, just want to document the "why" since the CT app is now reacing in to the LMS one in a tightly coupled way.

Reply:
per @shafqatfarhan :

No particular reason, I just felt the redundancy of yet another signal and task. In future, in an era of cross LOBs, this will become a filter that will either:
* Call the relevant service's deactivation endpoint
* Could be handled from AWS to notify messages to each service's upon SDN hit